### PR TITLE
Add delete comment function 

### PR DESCRIPTION
**Pull request by HelenaSG**
There's a new line at the end of each Java file, for example, 40 lines in total for DeleteCommentServlet.java. But it doesn't show up in Git if I view it in "File changed." I wonder whether you can see it when reviewing.

### CODE REVIEW

**Review by arjun-garg-google — APPROVED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java [hunk 1]
```diff
@@ -36,16 +37,23 @@
...
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    
-    //Load the entity by kind 
+
+    int max = 3;
+    int userChoice = getUserChoice(request);
+    if (userChoice != -1) {
+        max = userChoice;
+    }
+    System.out.println("max"+max);
```
Comment (gwroblew) [inline portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java]: These debug calls should be removed from the final code.

File: portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java [hunk 2]
```diff
@@ -36,16 +37,23 @@
...
+
     Query query = new Query("Comments").addSort("timestamp", SortDirection.DESCENDING);
 
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-    PreparedQuery results = datastore.prepare(query);
-
+    //PreparedQuery results = datastore.prepare(query);
```
Comment (gwroblew) [inline portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java]: Not needed anymore.

File: portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java [hunk 3]
```diff
@@ -36,16 +37,23 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    
-    //Load the entity by kind 
+
+    int max = 3;
+    int userChoice = getUserChoice(request);
```
Comment (gwroblew) [inline portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java:42]: userChoice is not very descriptive. commentCount, maxLimit, or maxCount would be better.

Comment (HelenaSG) [reply portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java:42]: Thank you! I will change it.

Comment (HelenaSG) [reply portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java:42]: > userChoice is not very descriptive. commentCount, maxLimit, or maxCount would be better.

Thank you! I will change it.

**Pull request merged**